doc/ref_model/chapters/chapter09.md

```diff
@@ -306,3 +318,137 @@ Topics include:
+
+Ideally, observability data would be sent from the source systems always to the same destination. This is achieved by using message brokers to decouple data producers from data consumers, and that inherently means that sources send data always to the same point. It is worth noting that message brokers can be considered as a common data bus.
+
+<a name="9.6.1.5"></a>
+#### 9.6.1.5. Which protocol, interface, and format to use
+
+While protocols and interfaces are dictated by the selection of the message broker (common data bus) system, data format is usually customizable according to the needs of users. The concept of Schema Registry mechanism, well known in the world of big data, is helpful here to make sure that message structures and formats are consistently used.
+
+<a name="9.6.2"></a>
+### 9.6.2. The Architecture
+
+In geographically dispersed large cloud deployments, a given telco cloud may have several cloud infrastructure components as well a large set of virtualized workloads (VNF/CNFs). It is important to monitor all of these workloads and infrastructure components. Furthermore, it is even more important to be able to correlate between the metrics provided by these entities to determine the performance and/or issues in such deployments. 
+
+The cloud deployment tends to shrink and expand based upon the customer demand. Therefore, an architecture is required that can scale on demand and does not force a strong tie between various entities. This means, the workloads and cloud infrastructure components that provide telemetry/performance metrics must not be burdened to discover each other. The capacity (e.g. speed, storage) of one component must not force overrun or underrun situations that would cause critical data to be lost or delayed to a point to render it useless. 
+
+Operators in charge of the Cloud Infrastructure (physical infra plus virtualization platform) require very detailed alarms and metrics to efficiently run their platform. While they need indicators about how well or poorly individual virtual machines and containers run, they don’t need a view inside these workloads. In fact, what and how workloads do is none of the NFVI operators business. The architecture must allow for different consumers to grant or deny access to available resources.
+
+Multiple workloads or network services can be deployed onto one or more sites. These workloads require logical separation so that their metrics don’t mix by accident or simply based on security and privacy requirements. This is achieved by deploying these workloads within their own tenant space. All virtualization platforms offer such isolation down to virtual networks per tenant.
+
+<a name="9.6.2.1"></a>
+#### 9.6.2.1. Push Vs. Pull
```

Comment:
I would suggest to describe Push, Pull and Brokering as 3 different methods or architectural constructs.
Now Push is mostly describing Brokering which in reality is a combination of services offering both Push and Pull and an architecture that normally combines the best of the two basic methods allowing independence in between producers and consumers.

Reply:
@TFredberg - Push and Pull are the methods (or techniques) where as brokering is facilitating component. The very next section does talk about message broker